sdk/synapse/Azure.Analytics.Synapse.AccessControl/src/Azure.Analytics.Synapse.AccessControl.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Synapse Analytics Access Control client library</Description>
     <AssemblyTitle>Azure.Analytics.Synapse.AccessControl</AssemblyTitle>
-    <Version>1.0.0-preview.1</Version>
+    <Version>1.0.0-preview.2</Version>
```

Comment:
This should have happened automatically.  I'm guessing it didn't because we don't have a live test pipeline setup yet.  (In the future, after a release our tooling will automatically bump all the versions to the next preview so our repos are never in a state where an accidental release would cause serious headaches.)

Reply:
We will setup live test pipeline in future. Created a GitHub issue to track this: https://github.com/Azure/azure-sdk-for-net/issues/14740

Reply:
Also, these PRs have to be merged: https://github.com/Azure/azure-sdk-for-net/pull/14751